firmware/shared_main.h

```diff
@@ -13,7 +13,6 @@ void note_on(uint8_t midi_note, uint8_t velocity, bool enabled) {
     dc1.amplitude(velocity / 127.); // DC amplitude controls filter env amount.
     osc_pulse_midi_note = midi_note;
     osc_pulse_target_frequency = (int)midi_note_to_frequency(midi_note);
-    osc_pulse.frequency(osc_pulse_frequency);
```

Comment:
This is removed in `brains2` which should be okay, since we're calling `osc_pulse.frequency()` here: https://github.com/datomusic/duo-imxrt/blob/valter/share-main/firmware/Synth.h#L197